### ISSUE TITLE
Allow forwarding private IRC queries with a config switch

### ISSUE THREAD

**Issue by ptrcnull** [42wim/matterbridge#1984]
**Is your feature request related to a problem? Please describe.**
I wanted to bridge a Telegram bot chat to my IRC bot - while messages to the IRC bot were forwarded, the replies were silently ignored. Only after reading some issues from 2016 I've learned that private queries are unconditionally ignored, even if specified in a gateway's `inout`

**Describe the solution you'd like**
Add an option to IRC config to handle private queries

**Describe alternatives you've considered**
Check if the target IRC username is used in some gateway? I'm not 100% sure how it works

**Additional context**
none

**Comment by 42wim** [comment]
duplicate of #191

**Comment by ptrcnull** [comment]
I've read the discussion under the linked issue and I feel like it doesn't quite match the usecase presented here and is a bit too broad in general
